.github/workflows/test_fedora.yml

```diff
@@ -0,0 +1,47 @@
+name: Fedora
+
+on:
+  push:
+    branches:
+      - main
+      - release
+  pull_request:
+    branches:
+      - main
+      - release
+
+jobs:
+  testFedora:
+      fail-fast: false
+    runs-on: ubuntu-latest
+    container:
+      image: fedora-rawhide
+    steps:
+      - name: Install Packages (dnf)
+        run: |
+          dnf -y update
+          dnf -y install adobe-source-sans-pro-fonts hunspell-en hunspell-en-GB hunspell-en-US
+          dnf -y install enchant2-devel hunspell-devel qt6-qttools-devel qt6-qtsvg-devel sed
+          dnf -y install python3-devel
+      - name: Checkout Source
+        uses: actions/checkout@v4
+      - name: Install Dependencies (pip)
+        run: |
+          pip install -U -r requirements.txt -r tests/requirements.txt
+      - name: Fedora specific modifications
+        run: |
+          sed -i 's/self.spellLanguage = "en"/self.spellLanguage = "en_US"/g' novelwriter/config.py
+          sed -i 's/spellcheck = en/spellcheck = en_US/g' tests/reference/baseConfig_novelwriter.conf
```

Comment:
Are these still needed? The spell check test now uses a mock dictionary, so it doesn't depend on any being installed.